### PR TITLE
09-MFA

### PR DESCRIPTION
**Pull request by volbap**
- Added `09-mfa.md` quickstart.
- Updated `index.yml`.

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by tenmaster** [comment]
+1 @mpaktiti @ntotten @chenkie

File: articles/native-platforms/ios-swift/09-mfa.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: MFA
+description: This tutorial will show you how to configure Multi Factor Authentication (MFA) via Google Authenticator in your app.
+---
+
```
Comment (mpaktiti) [inline articles/native-platforms/ios-swift/09-mfa.md:5]: @pablolvillar missing System Requirements and Get Seed Project sections

File: articles/native-platforms/ios-swift/09-mfa.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+### 1. Enable MFA in your account
+
+You have to enable the MFA feature in your account. 
+
+First, follow [this link](https://manage.auth0.com/#/multifactor). Then, turn the switch on for **Google Authenticator**, under the *Choose a Provider* section.
```
Comment (mpaktiti) [inline articles/native-platforms/ios-swift/09-mfa.md]: @pablolvillar replace link `[this link](https://manage.auth0.com/#/multifactor)` with `[this link](${uiURL}/#/multifactor)`

https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#document-variables

**Comment by mpaktiti** [comment]
Seed project (to be deployed before this PR): https://github.com/auth0-samples/auth0-ios-swift-sample/pull/12

File: articles/native-platforms/ios-swift/09-mfa.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+If you want to use MFA in **all** of your clients, the easiest you can do is disabling this conditional in the script:
+
+```javascript
+if (CLIENTS_WITH_MFA.indexOf(context.clientID) !== -1)
+```
+
+### Done!
+
+Peace of cake, wasn't it? Now you should get a google authenticator screen when you try to login with social providers in your app using [Lock-iOS](https://github.com/auth0/Lock.iOS-OSX).
```
Comment (sebacancinos) [inline articles/native-platforms/ios-swift/09-mfa.md]: Piece of cake

Comment (sebacancinos) [inline articles/native-platforms/ios-swift/09-mfa.md]: Google Authenticator should be capitalised, I think.

Comment (tenmaster) [inline articles/native-platforms/ios-swift/09-mfa.md]: Peace > Piece 
@pablolvillar

Comment (volbap) [inline articles/native-platforms/ios-swift/09-mfa.md]: Yeah let's make peace here 🕊☮ . Thanks 😜

File: articles/native-platforms/ios-swift/09-mfa.md (hunk 1 above)
Comment (volbap) [inline articles/native-platforms/ios-swift/09-mfa.md:5]: Added.

**Comment by volbap** [comment]
@mpaktiti Done with the fixes.

**Comment by mpaktiti** [comment]
Before deployment:
- Seed project should be merged https://github.com/auth0-samples/auth0-ios-swift-sample/pull/12
- Sample is at auth0-samples repo, download zip functionality should be in place.
- `index.yml` should be checked again

**Comment by volbap** [comment]
Closing this pull request. These changes will be included in a new pull request.

**Pull request closed without merging**